### DISCUSSION TITLE
Link with multiple languages available, where does it go

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#251] [Q&A] [answered]
I have here a version of BabyTV, included on the Polsat pack of Astra channels.

There's a twist to it, I got all the languages from all the countries BabyTV streams in, and they seem to be all working.

![image](https://user-images.githubusercontent.com/30985701/121821597-42a3e800-cc9a-11eb-9e57-e4120b25c350.png)

Where should it go ?

**Comment by krm28-s** [comment]
👍👍👍👍i need m3u bro please

Von meinem iPhone gesendet

**Comment by Dum4G** [comment]
To the origin country playlist as we usually do. Just don't specify language. We can type "Selectable" in tvg-language as long as @freearhey allows us

**Reply by BellezaEmporium** [reply]
That's what I thought, but due to the "special" nature of this, just wanted to make sure we stick to the rules

**Comment by freearhey** [comment]
Do you only have one such link or are there more?

**Reply by BellezaEmporium** [reply]
For the moment, only this one. The other links that I've encountered didn't go as far as 2 audios (often original audio + other one)

**Reply by freearhey** [reply, marked as answer]
In this case I think it will be enough to list all languages in the `tvg-language`.
